examples/music-licensing-challenge/src/app/api/movies.py

```
@@ -0,0 +1,24 @@
+from ..db.database import get_db
+from ..models.movie import Movie
+from ..schemas.movies import MovieWithAllData
+from ..repository.movies import MovieRepository
+
+router = APIRouter()
+
+
+@router.get("/", response_model=List[MovieWithAllData] | MovieWithAllData)
+def read_movies(
+    id: Optional[str] = None,
+    db: Session = Depends(get_db),
```

Ruff (0.8.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)